docs/developer/kiosk.md

```diff
@@ -0,0 +1,106 @@
+# Kiosk
+
+To allow users to showcase their games in a fun and unique way, the games' Kiosk is added to Makecode Arcade. The purpose of the Kiosk is to further encourage users to upload their games. The Kiosk is an interface where a list of games is displayed in a carousel, letting a user scroll through what's currently highlighted to play. Hopefully, this will users excite users to want to upload their games.  
+
+## Getting started 
+
+The default Kiosk contains an initial set some of our most popular games from Arcade. This page is located at https://arcade.makecode.com/kiosk.
+
+![Starting screen](/static/developer/kiosk/select-a-game.png)
+
+## Clearing the Kiosk
+
+To have students start with an empty (clean) Kiosk, so that they won't be tempted to just strart by playing random games, you can add the clean flag to the url:
```

Comment:
```suggestion
To have students start with an empty (clean) Kiosk, so that they won't be tempted to play random games, you can add the clean flag to the url:
```